applications/GeoMechanicsApplication/custom_elements/U_Pw_small_strain_element.cpp

```diff
@@ -176,31 +167,27 @@ void UPwSmallStrainElement<TDim,TNumNodes>::InitializeSolutionStep(const Process
     // reset hydraulic discharge
     this->ResetHydraulicDischarge();
 
-    KRATOS_CATCH("");
+    KRATOS_CATCH("")
 }
 
-//----------------------------------------------------------------------------------------------------
 template< unsigned int TDim, unsigned int TNumNodes >
-void UPwSmallStrainElement<TDim,TNumNodes>::
-    ResetHydraulicDischarge()
+void UPwSmallStrainElement<TDim,TNumNodes>::ResetHydraulicDischarge()
```

Comment:
Even more nitpicking:
```suggestion
void UPwSmallStrainElement<TDim, TNumNodes>::ResetHydraulicDischarge()
```

Reply:
done